tests/functional/install/basic.bats

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu Vinod
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+global_setup() {
+
+    create_testworking_dir
+    sudo qemu-img create -f raw "$TESTWORKINGDIRECTORY"/testimgfile.img 5G
```

Comment:
Since we had to run this as root, why do we need the sudo?

Reply:
done